### PR TITLE
New package: kvmtool-3.18.0

### PR DESCRIPTION
**Pull request by auser509**

### CODE REVIEW

**Comment by maxice8** [comment]
Just amend commits and force push, don't close and re-open PRs

File: srcpkgs/kvmtool/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Template file for 'kvmtool'
+pkgname=kvmtool
+version=3.18.0
+revision=1
+_githash=41d773e2070aabd3c4ff20c059f45c3e9a0f9d20
+wrksrc="${_githash}"
+build_style=gnu-makefile
+make_use_env=no
```
Comment (Gottox) [inline srcpkgs/kvmtool/template]: Any string != "" will **enable** `make_use_env`. `=no` is very missleading as it actually *enables* it.

Comment (auser509) [reply srcpkgs/kvmtool/template]: Indeed, thanks. Pushed a fix. I noticed that quite a few templates use `make_use_env=compliant`, which may also be confusing (this isn't documented and as you properly point out makes no difference). Other templates do `make_use_env=1` which looks better, we might though like to explicitly mention this in the Manual in order to make things more consistent?

File: srcpkgs/sparse/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+make_use_env=yes
+short_desc="Semantic Parser for C"
+maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
+license="MIT"
+homepage="https://git.kernel.org/pub/scm/devel/sparse/sparse.git"
+distfiles="${homepage}/snapshot/${pkgname}-${version}.tar.gz"
+checksum=5ca5b7be7240a97fbdc6194f18d64837f9af070aeb0c3052f0a5d014acf1d048
+
+pre_build() {
+	make_build_args="CC=$CC LD=$CC AR=$AR"
```
Comment (maxice8) [inline srcpkgs/sparse/template]: this is a global variable, put it under build_style=

**Comment by jnbr** [comment]
Pushed wrong branch?

**Comment by maxice8** [comment]
ping, this seems to be the wrong package.

**Pull request closed without merging**